test/testdata/resolver/invalid_alias.rb.symbol-table-raw.exp

```diff
@@ -0,0 +1,24 @@
+class <C <U <root>>> < <C <U Object>> () @ (... removed core rbi locs ..., Loc {file=https://github.com/sorbet/sorbet/tree/master/bazel-out/host/genfiles/rbi/procs.rbi start=1:1 end=252:4}, Loc {file=test/testdata/resolver/invalid_alias.rb start=3:1 end=15:4})
+  class <S <C <U <root>>> $1>[<C <U <AttachedClass>>>] < <S <C <U Object>> $1> ()
+    method <S <C <U <root>>> $1><N <U <static-init>> $CENSORED> (<blk>) @ Loc {file=test/testdata/resolver/invalid_alias.rb start=3:1 end=15:4}
+      argument <blk><block> @ Loc {file=test/testdata/resolver/invalid_alias.rb start=??? end=???}
+  class <C <U BadAliasingClassMethod1>> < <C <U Object>> () @ Loc {file=test/testdata/resolver/invalid_alias.rb start=3:1 end=3:30}
+  class <S <C <U BadAliasingClassMethod1>> $1>[<C <U <AttachedClass>>>] < <S <C <U Object>> $1> () @ Loc {file=test/testdata/resolver/invalid_alias.rb start=4:3 end=4:8}
+    type-member(+) <S <C <U BadAliasingClassMethod1>> $1><C <U <AttachedClass>>> -> LambdaParam(<S <C <U BadAliasingClassMethod1>> $1><C <U <AttachedClass>>>, lower=T.noreturn, upper=BadAliasingClassMethod1) @ Loc {file=test/testdata/resolver/invalid_alias.rb start=3:7 end=3:30}
+    method <S <C <U BadAliasingClassMethod1>> $1><U <static-init>> (<blk>) @ Loc {file=test/testdata/resolver/invalid_alias.rb start=3:1 end=11:4}
+      argument <blk><block> @ Loc {file=test/testdata/resolver/invalid_alias.rb start=??? end=???}
+    method <S <C <U BadAliasingClassMethod1>> $1><U hello> (<blk>) @ Loc {file=test/testdata/resolver/invalid_alias.rb start=5:5 end=5:14}
+      argument <blk><block> @ Loc {file=test/testdata/resolver/invalid_alias.rb start=??? end=???}
+  class <S <S <C <U BadAliasingClassMethod1>> $1> $1>[<C <U <AttachedClass>>>] < <S <S <C <U Object>> $1> $1> () @ Loc {file=test/testdata/resolver/invalid_alias.rb start=4:3 end=4:8}
+    type-member(+) <S <S <C <U BadAliasingClassMethod1>> $1> $1><C <U <AttachedClass>>> -> LambdaParam(<S <S <C <U BadAliasingClassMethod1>> $1> $1><C <U <AttachedClass>>>, lower=T.noreturn, upper=AppliedType {
+  klass = <S <C <U BadAliasingClassMethod1>> $1>
+  targs = [
+    <C <U <AttachedClass>>> = BadAliasingClassMethod1
+  ]
+}) @ Loc {file=test/testdata/resolver/invalid_alias.rb start=4:3 end=4:8}
```

Comment:
I would love to not print multi-line types in this single-line symbol-table format... 🤔

Reply:
i should note: this is very non-blocking.

Reply:
#1960